web/src/app/util/FilterContainer.js

```diff
@@ -62,7 +70,14 @@ export default class FilterContainer extends React.PureComponent {
       <React.Fragment>
         <IconButton
           color='inherit'
-          onClick={e => this.setState({ anchorEl: e.target })}
+          onClick={e =>
+            this.setState({
+              anchorEl: e.target,
+              ariaExpanded: !this.state.ariaExpanded,
+            })
+          }
+          title='filter'
+          aria-expanded={this.state.ariaExpanded}
```

Comment:
you can set this to whether or not `anchorEl` is set, as we do on line 117 here

Reply:
For this, we need to keep it because anchorEl uses null and not bool values.